Cloud Enterprise/Getting Started Examples/aws/terraform/README.md

```diff
@@ -2,12 +2,16 @@
 
 ## Details
 
-This tree provides a basic test bed for an installation of [Elastic Cloud Enterprise](https://www.elastic.co/cloud/enterprise) on 2 servers.
+This tree provides a basic example installation of [Elastic Cloud Enterprise](https://www.elastic.co/cloud/enterprise) on 3 servers using terraform & ansible.
 
 Note that the terraform files and installation script are intentionally basic.
 
 A full production deployment should make use of auto scaling groups, load balancers and other high-availability constructs which have been left out of this setup. See [the Elastic Cloud Enterprise planning docs](https://www.elastic.co/guide/en/cloud-enterprise/1.1/ece-planning.html) for additional details regarding production planning and deployment.
 
+## Prereqs on your machine
+- Terraform v0.11.X
+- Ansible
```

Comment:
Would it be better to use [ansible-runner](https://ansible-runner.readthedocs.io/en/latest/standalone.html) on the target machines so we have a single dependency only on the users machine ( and one that is somewhat easier to manage since it's a binary download) ?
Having ansible locally might make sense for a fleet, but for the sake of the example the runner might be slightly better. Anyway, it's fine either way, just a suggestion, it's not meant to hold the PR up.

Reply:
I've thought of that path initially, there are few problems with it - 
1. You'll need to install ansible on all 3 servers (or otherwise split sever creation for 'installer' and 'others'
2. Will make this OS specific (since you'll need to write user-data to install ansible on the 'installer' machine. Technically you can now use this example and replace ubuntu AMI with centos AMI pretty easily.

Reply:
Makes sense. We could think of providing a wrapper script together with the ansible role, not as an example, that would install ansible runner on the supported OS-es and call the role locally. I think it might be useful for  anyone who only cares about the results and not ansible itself, the process would be simpler.